packages/react/src/components/ItemLayout/ItemLayout.tsx

```diff
@@ -101,8 +99,6 @@ class ItemLayout extends UIComponent<ReactProps<ItemLayoutProps>, any> {
     contentCSS: PropTypes.object,
     contentMediaCSS: PropTypes.object,
     endMediaCSS: PropTypes.object,
-    truncateContent: PropTypes.bool,
```

Comment:
If we just take these out, won't they just get passed through to html and not do anything.  Should we leave as warning / compile error or something for a few releases before we actually remove?

Reply:
This is already marked as a BREAKING CHANGE in `CHANGELOG.md` and it points to the description of the PR where we have a mitigation path. This should be enough for our clients to update without too much friction